scripts/Logistics-Pipes.zs

```diff
@@ -0,0 +1,873 @@
+
+// --- Provider Logistics Pipe
+recipes.remove(<LogisticsPipes:item.PipeItemsProviderLogistics>);
+
+// --- Provider Logistics Pipe MK2
+recipes.remove(<LogisticsPipes:item.PipeItemsProviderLogisticsMk2>);
+
+// --- Crafting Logistics Pipe
+recipes.remove(<LogisticsPipes:item.PipeItemsCraftingLogistics>);
+
+// --- Crafting Logistics Pipe MK2
+recipes.remove(<LogisticsPipes:item.PipeItemsCraftingLogisticsMk2>);
+
+// --- Crafting Logistics Pipe MK3
+recipes.remove(<LogisticsPipes:item.PipeItemsCraftingLogisticsMk3>);
+
+// --- Satellite Logistics Pipe
+recipes.remove(<LogisticsPipes:item.PipeItemsSatelliteLogistics>);
+
+// --- Supplier Logistics Pipe
+recipes.remove(<LogisticsPipes:item.PipeItemsSupplierLogistics>);
+
+// --- Remote Order Logistics Pipe
+recipes.remove(<LogisticsPipes:item.PipeItemsRemoteOrdererLogistics>);
+
+// --- Logistics System Entrance Pipe
+recipes.remove(<LogisticsPipes:item.PipeItemsSystemEntranceLogistics>);
+
+// --- Logistics System Entrance Pipe
+recipes.remove(<LogisticsPipes:item.PipeItemsSystemDestinationLogistics>);
+
+// --- Logistics Inventory System Connector
+recipes.remove(<LogisticsPipes:item.PipeItemsInvSysConnector>);
+
+
+// ||||| Fluid Pipes |||||
+
+
+// --- Logistics Basic Fluid Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidBasic>);
+
+// --- Logistics Fluid Container Supplier
+recipes.remove(<LogisticsPipes:item.PipeItemsFluidSupplier>);
+
+// --- Logistics Fluid Supplier
+recipes.remove(<LogisticsPipes:item.PipeFluidSupplierMk2>);
+
+// --- Logistics Fluid Provider Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidProvider>);
+
+// --- Logistics Fluid Request Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidRequestLogistics>);
+
+// --- Logistics Fluid Satellite Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidSatellite>);
+
+// --- Logistics Fluid Insertion Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidInsertion>);
+
+// --- Logistics Fluid Extractor Pipe
+recipes.remove(<LogisticsPipes:item.PipeFluidExtractor>);
+
+
+// ||||| Chassis |||||
+
+
+// --- Logistics Chassi MK1
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk1>);
+
+// --- Logistics Chassi MK2
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk2>);
+
+// --- Logistics Chassi MK3
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk3>);
+
+// --- Logistics Chassi MK4
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk4>);
+
+// --- Logistics Chassi MK5
+recipes.remove(<LogisticsPipes:item.PipeLogisticsChassiMk5>);
+
+
+
+// *======= Adding Back Recipes =======*
+// --- Assembler Recipes --- 
+
+// - LV Tier
+// ||||| Blocks |||||
+// --- Power Junction
+Assembler.addRecipe(<LogisticsPipes:logisticsSolidBlock:1>, [<gregtech:gt.blockmachines:21>, <LogisticsPipes:item.PipeItemsBasicLogistics> * 4, <ore:circuitBasic> * 2, <ore:cableGt04AnyCopper> * 4, <ore:screwBlackSteel> * 4, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.lead> * 1152, 800, 30);

```

Comment:
Why black steel? it can't be made without EBF, so the whole LP is locked behind it as a consequence.

Reply:
Changed to steel